landing-page/landing-page.css

```diff
@@ -0,0 +1,238 @@
+body{
+    background-color: var(--bg-color);
+}
+.page-container{
+    display: flex;
+    flex-direction: column;
+    height: 100vh;
+}
+.navbar{
+    margin-bottom: 5px;
+    padding: 1.3rem var(--spacing3) ;
+    display: flex;
+    align-items: center;
+    justify-content: center;
+    padding-bottom: 6px;
```

Comment:
You could use the 3 valued short-hand for padding here. That goes like this 
`padding: [padding-top]   [padding-left-right]   [padding-bottom]`

In your case it would make
`padding: 1.3rem var(--spacing3) 6px`

This does the work in a single line.